Library/Formula/gnupg2.rb

```diff
@@ -69,7 +67,7 @@ def install
 
     # Conflicts with a manpage from the 1.x formula, and
     # gpg-zip isn't installed by this formula anyway
-    rm man1/"gpg-zip.1"
+    rm_f man1/"gpg-zip.1"
```

Comment:
Thanks, I'd actually meant to ask you or Jack whether this was helpful or not so the feedback on that is great, Appreciated. Will stop doing it :smiley_cat: